src/apic-extension/azext_apic_extension/tests/latest/constants.py

```diff
@@ -7,4 +7,8 @@
 
 TEST_REGION = "eastus"
 # to set USERASSIGNED_IDENTITY, refer to https://learn.microsoft.com/en-us/azure/api-center/import-api-management-apis?tabs=portal#option-2-import-apis-directly-from-your-api-management-instance
-USERASSIGNED_IDENTITY = os.getenv('USERASSIGNED_IDENTITY')
\ No newline at end of file
+USERASSIGNED_IDENTITY = os.getenv('USERASSIGNED_IDENTITY')
+# aws credentials KeyVault references
+ACCESS_KEY_LINK = "https://kv-canary-franktest.vault.azure.net/secrets/AccessKey"
```

Comment:
should we put these key links in github secrets?

Reply:
Yes, let's do it. Thank you.